library/src/main/java/software/amazon/location/auth/utils/AwsRegions.kt

```diff
@@ -0,0 +1,45 @@
+package software.amazon.location.auth.utils
+
+
+enum class AwsRegions(val regionName: String) {
+    GovCloud("us-gov-west-1"),
```

Comment:
This may be something that we have to update over time as new regions are added.